common/thorhelper/thorsoapcall.cpp

```diff
@@ -1766,7 +1766,7 @@ class CWSCAsyncFor : implements IWSCAsyncFor, public CInterface, public CAsyncFo
             tail = "Dataset/Row";
 
         CMatchCB matchCB(*this, url, tail, meta);
-        Owned<IXMLParse> xmlParser = createXMLParse((const void *)response.str(), (unsigned)response.length(), path.str(), matchCB, options, false);
+        Owned<IXMLParse> xmlParser = createXMLParse((const void *)response.str(), (unsigned)response.length(), path.str(), matchCB, options, master->flags&SOAPFusescontents);
```

Comment:
picky: visual studio will complain about an implicit conversion from unsigned to bool.  Add a comparison against 0 to avoid it.

(master->flags&SOAPFusescontents) != 0